.gitignore

```diff
@@ -1,4 +1,4 @@
-/target
+target/
```

Comment:
Main goal was to ignore "dashboard-builder-bom/target". So is it OK if I will change it to:

```
/target
/dashboard-builder-bom/target
```